package.json

```diff
@@ -9,13 +9,14 @@
     "node": ">=12.13"
   },
   "scripts": {
-    "docs": "yarn --cwd packages/docs start",
+    "docs": "yarn --cwd packages/docs start --no-clear-screen",
     "icons": "yarn --cwd packages/odyssey-icons svgo",
     "lerna-version": "lerna version --no-git-tag-version --force-publish",
     "prelerna-publish": "lerna run build",
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry"
   },
   "devDependencies": {
+    "eslint": "^7.9.0",
```

Comment:
Since this is specified at the root of all our dependencies, I'd suggest adding a `lint` script (and our configuration) to our root `package.json`. That way all of our JS code adheres to the same rule-set!